README.md

```diff
@@ -72,28 +85,96 @@ This template follows a very simple project structure:
 - `index.js`: Entry point of your application as per React-Native standards.
 
 ## Splash screen customization
+
 To customize the splash screen (logo and background color) use the CLI provided in the [official docs](https://github.com/zoontek/react-native-bootsplash#assets-generation).
 
 ## Setup environments
+
+### Using scripts from console
+
+The template allready has scripts to excecute the project calling an especific environment defined into the package.json file. Keep in mind that if you are going to create new `envs` you have to define the script to build the project propertly.
+
 Modify the environment variables files in root folder (`.env.development`, `.env.production` and `.env.staging`)
 
```

Comment:
@AGVasquez What do you think about adding another section here of how to create an archive for iOS and an APK for android? Maybe explaining the actions from xcode (select schema) and from console `./gradle ....`